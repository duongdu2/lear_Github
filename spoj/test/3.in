6
3
2 32 43 
6
66 52 39 30 33 42 
8
59 34 77 63 38 29 84 24 
7
9 67 27 6 14 88 14 
5
22 82 40 48 49 
4
63 33 79 59 
