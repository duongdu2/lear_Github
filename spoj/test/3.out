77
160
224
182
179
201
